docs/networking/ipv6.md

```diff
@@ -0,0 +1,37 @@
+# IPv6
+
+{{ kops_feature_table(kops_added_ff='1.22') }}
+
+kOps has experimental support for configuring clusters with IPv6-only pods and dual-stack nodes.
```

Comment:
Dual-stack should probably not get mentioned.
```suggestion
kOps has experimental support for configuring clusters with IPv6-only pods on AWS instances with IPv6 enabled.
```

Reply:
The nodes are dual-stack.